File: solutions/beecrowd/1013/1013.js
const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
});

(() => {
  rl.on('line', (line) => {
    const [a, b, c] = line.split(' ').map(x => parseInt(x))

    let m = Math.floor((a + b + Math.abs(a - b)) / 2)
    m = Math.floor((m + c + Math.abs(m - c)) / 2)

    console.log(`${m} eh o maior`)
  })
})()

File: solutions/beecrowd/1013/1013.dart
import 'dart:io';

void main() {
  var line;
  double a, b, c, m;

  while ((line = stdin.readLineSync()) != null) {
    var numbers = line.split(' ');
    a = double.parse(numbers[0]);
    b = double.parse(numbers[1]);
    c = double.parse(numbers[2]);

    m = (a + b + (a - b).abs()) / 2;
    m = (m + c + (m - c).abs()) / 2;

    print("${m.round()} eh o maior");
  }
}
